src/lib/ngsi/ContextElement.cpp

```diff
@@ -80,29 +80,29 @@ ContextElement::ContextElement(const std::string& id, const std::string& type, c
 */
 std::string ContextElement::render
 (
-  ApiVersion   apiVersion,
   bool         asJsonObject,
   RequestType  requestType,
   bool         comma,
   bool         omitAttributeValues
 )
 {
+#if 0
   std::string  out                              = "";
-  bool         attributeDomainNameRendered      = attributeDomainName.get() != "";
+  bool         attributeDomainNameRendered      = false; // FIXME PR
   bool         contextAttributeVectorRendered   = contextAttributeVector.size() != 0;
-  bool         domainMetadataVectorRendered     = domainMetadataVector.size() != 0;
+  bool         domainMetadataVectorRendered     = false; // FIXME PR
 
-  bool         commaAfterDomainMetadataVector   = false;  // Last element
   bool         commaAfterContextAttributeVector = domainMetadataVectorRendered;
   bool         commaAfterAttributeDomainName    = domainMetadataVectorRendered  || contextAttributeVectorRendered;
   bool         commaAfterEntityId               = commaAfterAttributeDomainName || attributeDomainNameRendered;
+#endif
```

Comment:
All this to be removed in this PR, right?

Reply:
Yes. I missed it...

Reply:
Fixed in 6ddd2cb76fcdbc76cc452b02d631c9b1143b5b71